tensorrt_llm/llmapi/llm.py

```diff
@@ -112,7 +113,11 @@ def __init__(self,
         try:
             self.pytorch_backend_config = kwargs.pop('pytorch_backend_config',
                                                      None)
-            self.args = LlmArgs.from_kwargs(
+
+            llm_args_cls = TorchLlmArgs if kwargs.get(
```

Comment:
This is a knit-pick, but couldn't this be put in a dictionary or a factory class?

Reply:
I see, it could be fine as there are only two backends, we can change it like you mentioned if we have three or more backends in the future.

Reply:
Sounds good -- yeah, I just wanted to call it out. :)